src/weather-bar.ts

```diff
@@ -299,6 +319,11 @@ export class WeatherBar extends LitElement {
       background-color: var(--color-exceptional);
       color: var(--color-exceptional-foreground, var(--primary-text-color));
     }
+    .bar-span {
+      border-style: solid;
+      border-width: 1px;
+      border-color: transparent
```

Comment:
Does this impact the rendering when there are no borders requested? We may have to only set the border width when borders are configured to avoid adding a 1px space around color blocks by default.